client/selectors/index.js

```diff
@@ -78,6 +78,10 @@ export const getPlanningItemReadOnlyState = (state) => get(state, 'planning.read
 export const getEventReadOnlyState = (state) => get(state, 'events.readOnly')
 export const getSessionDetails = (state) => get(state, 'session')
 export const getCurrentUserId = (state) => get(state, 'session.identity._id')
+export const getEditorOpened = (state) => get(state, 'assignment.editorOpened')
```

Comment:
Can getEditorOpened be renamed to 'getAssignmentEditorOpened' please ? As we already have event and planning editors.